data/raw/usa/usa_policy_data_sources.csv

```diff
@@ -12,630 +42,830 @@ USA,all,all,all,all,2/23/20,travel_ban_intl_out,1,"Iran, Ialy",NA,NA,Y,travel wa
+USA,North Carolina,NC,all,all,3/10/20,social_distance,0.2,NA,NA,NA,Y,recommended to maintain distance,https://governor.nc.gov/news/governor-cooper-declares-state-emergency-respond-coronavirus-covid-19,3/26/20
+USA,North Carolina,NC,all,all,3/10/20,work_from_home,0.25,NA,NA,NA,Y,"Governor Cooper declared a state of emergency, which actives the Emergency Operation Center, and allows for control of cost burdens, protects consumers from price gouging, and makes it easier to purchase necessary medical supplies. The executive order also makes other recommendations around telework schedules, social gatherings, school closures, and high-risk populations, such as those in adult living facilities.",https://governor.nc.gov/news/governor-cooper-declares-state-emergency-respond-coronavirus-covid-19,3/17/20
+USA,North Carolina,NC,all,all,3/13/20,testing_regime,1,NA,NA,NA,N,Eliminated the Member Cost-share for COVID-19 testing to ensure there are no cost barriers to testing.,https://www.nctreasurer.com/news/press-releases/2020/03/13/state-health-plan-board-approves-emergency-authority-respond-covid-19,3/16/20
+USA,North Carolina,NC,all,all,3/15/20,no_gathering,0.25,NA,NA,100,N,The executive order also prohibits mass gatherings of more than 100 people.,https://governor.nc.gov/news/governor-cooper-issues-executive-order-closing-k-12-public-schools-and-banning-gatherings-more,3/17/20
+USA,North Carolina,NC,all,all,3/16/20,school_closure,1,NA,NA,NA,N,all schools will be closed,https://www.nctreasurer.com/news/press-releases/2020/03/13/state-health-plan-board-approves-emergency-authority-respond-covid-19,3/16/20
+USA,North Carolina,NC,all,all,3/17/20,social_distance,0.4,NA,NA,NA,N,"which closes sit down services at restaurants and bars, a",https://files.nc.gov/governor/documents/files/EO118.pdf,3/26/20
+USA,North Carolina,NC,all,all,3/23/20,no_gathering,0.5,NA,NA,50,N,"Executive Order 120, which enforces tighter limits on gatherings (50-plus)",https://files.nc.gov/governor/documents/files/EO120.pdf,3/26/20
 USA,North Carolina,NC,all,all,3/23/20,school_closure,1,NA,NA,NA,N,"Executive Order 120, which extends school closures statewide to May 15",https://files.nc.gov/governor/documents/files/EO120.pdf,3/26/20
-USA,North Carolina,NC,all,all,3/23/20,no_gathering,1,NA,NA,50,N,"Executive Order 120, which enforces tighter limits on gatherings (50-plus)",https://files.nc.gov/governor/documents/files/EO120.pdf,3/26/20
-USA,North Carolina,NC,all,all,3/23/20,social_distance,1,NA,NA,NA,N,"Executive Order 120, which enforces closes some businesses that require close social interaction, and limits visitors to long-term care centers.",https://files.nc.gov/governor/documents/files/EO120.pdf,3/26/20
-USA,New Hampshire,NH,all,all,3/23/20,social_distance,1,NA,NA,NA,N,"Provide temporary authority perform secure remote online notarization. Temporarily authorize out-of-state pharmacies to act as licensed mail-order 
```

Comment:
hmm... do we know what happened here? Is this potentially an issue anywhere else as well?